test/integration/features/provision_various_versions.feature

```diff
@@ -0,0 +1,26 @@
+@provision-various-versions @openshift @minishift-only
+Feature: Provision all major Openshift versions
+  As a user I can provision major versions of openshift
+
+Scenario Outline:: Provision all major OpenShift versions
+    Given Minishift has state "Does Not Exist"
+     When executing "minishift start --openshift-version <serverVersion>" succeeds
+     Then Minishift should have state "Running"
+     When executing "minishift openshift version" succeeds
+     Then stdout should contain
+     """
+     openshift <serverVersion>
+     """
+      And JSON config file "machines/minishift.json" contains key "OcPath" with value matching "<ocVersion>"
+     When executing "minishift oc-env" succeeds
+     Then stdout should contain
+      """
+      <ocVersion>
+      """
+     When executing "minishift delete --force" succeeds
+     Then Minishift should have state "Does Not Exist"
+Examples:
```

Comment:
Should be indented by 2 spaces as it is part of Feature block.

Reply:
Examples: -> Examples: OpenShift and oc versions